src/engine/controls/loopingcontrol.cpp

```diff
@@ -782,7 +784,7 @@ void LoopingControl::notifySeek(double dNewPlaypos, bool adjustingPhase) {
             setLoopingEnabled(false);
         }
         if (currentSample <= loopSamples.end &&
-                dNewPlaypos > loopSamples.end) {
+                dNewPlaypos >= loopSamples.end) {
```

Comment:
Why is this change necessary?

Reply:
Say two hotcues (A and B) are 4 bars apart from each other. A 4 bar loop is active starting at hotcue A. The end of the loop is exactly at hotcue B. When hotcue B ist activated the loop should be deactivated and the position set to B. Without this change, the position is immediately set to the beginning of the loop because the loop is still active.

Does that clarify it?

Reply:
Yes, Thank you. Please explain this as a comment.

Reply:
By comment you mean in the PR or in the code?

Reply:
Yes, right.

Reply:
.. in the code ;)

Reply:
:man_facepalming:  sorry